.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,132 @@
+      image: node:10
+    env:
+      NPM_CONFIG_UNSAFE_PERM: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Cache Dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            package-lock.json
+            */*/node_modules
+          key: ${{ runner.os }}-node10-${{ hashFiles('**/package-lock.json') }}
+      - name: Install Root Dependencies
+        run: npm install --ignore-scripts
+      - name: Boostrap Dependencies
+        run: npx lerna bootstrap --no-ci
+      - name: Unit tests
+        run: npm run test
+  node12:
+    runs-on: ubuntu-latest
+    container:
+      image: node:12
+    env:
+      NPM_CONFIG_UNSAFE_PERM: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Cache Dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            package-lock.json
+            */*/node_modules
+          key: ${{ runner.os }}-node12-${{ hashFiles('**/package-lock.json') }}
+      - name: Install Root Dependencies
+        run: npm install --ignore-scripts
+      - name: Boostrap Dependencies
+        run: npx lerna bootstrap --no-ci
+        env:
+          NPM_CONFIG_UNSAFE_PERM: true
+      - name: Unit tests
+        run: npm run test
+      - name: Report Coverage
+        run: npm run codecov
+  node14:
+    runs-on: ubuntu-latest
+    container:
+      image: node:14
+    env:
+      NPM_CONFIG_UNSAFE_PERM: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Cache Dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            package-lock.json
+            */*/node_modules
+          key: ${{ runner.os }}-node14-${{ hashFiles('**/package-lock.json') }}
+      - name: Install Root Dependencies
+        run: npm install --ignore-scripts
+      - name: Boostrap Dependencies
+        run: npx lerna bootstrap --no-ci
+      - name: Unit tests
+        run: npm run test
+  node12-browsers:
+    runs-on: ubuntu-latest
+    container:
+      image: circleci/node:12-browsers
+    env:
+      NPM_CONFIG_UNSAFE_PERM: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Permission Setup
+        run: sudo chmod -R 777 /github /__w
+      - name: Cache Dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            package-lock.json
+            */*/node_modules
+          key: ${{ runner.os }}-node12-${{ hashFiles('**/package-lock.json') }}
+      - name: Install Root Dependencies
+        run: npm install --ignore-scripts
+      - name: Boostrap Dependencies
+        run: npx lerna bootstrap --no-ci
+        env:
+          NPM_CONFIG_UNSAFE_PERM: true
+      - name: Unit tests
+        run: npm run test
+      - name: Report Coverage
+        run: npm run codecov
```

Comment:
maybe add a new line at the end of the file for consistency

Reply:
Done